#!/usr/bin/python3
def raise_exception():
    t= "food"
    print('{:d}'.format(t))
